app/qml/gps/MMStakeoutPanel.qml

```diff
@@ -58,15 +58,26 @@ MMDrawer {
     root.open()
   }
 
+  Item {
+    focus: true
+
+    Keys.onPressed: {
+      if ( event.key === Qt.Key_Back || event.key === Qt.Key_Escape ) {
+        root.endStakeout()
+        event.accepted = true
+      }
+    }
+  }
+
+  closePolicy: Popup.NoAutoClose
```

Comment:
This should be part of MMDrawer, in case we want it. we want same behaviour for all drawers